bim2sim/tasks/bps/sb_2b_generation.py

```diff
@@ -43,11 +43,23 @@ def run(self, elements: dict) -> tuple[dict]:
         if not self.playground.sim_settings.close_space_boundary_gaps:
             return elements,
         try:
+            total_area_before_split = 0
             inst_2b = self._compute_2b_bound_gaps(elements)
+            for sb in inst_2b.values():
+                total_area_before_split += sb.bound_area
             CorrectSpaceBoundaries.split_non_convex_bounds(
                 CorrectSpaceBoundaries(self.playground),
                 inst_2b,
                 self.playground.sim_settings.split_bounds)
+            total_area_after_split = 0
+            for sb in inst_2b.values():
+                total_area_after_split += sb.bound_area
+            rel_dif_per = (abs(total_area_before_split-total_area_after_split)/
+                       total_area_before_split * 100)
+            if rel_dif_per > 1:
```

Comment:
```suggestion
            if rel_dif_per > 1e-3:
```
I would decrease the tolerance, as it is just a warning. Usually, splitting should be performed without surface area loss (rel_diff_per == 0) and a low tolerance of 1e-3 would account for numerical issues, but detect even small changes in surface area.